src/js/webrequest.js

```diff
@@ -158,20 +159,14 @@ function onBeforeSendHeaders(details) {
     // This will only happen if the above code sets the action for the request
     // to block
     if (requestAction == constants.BLOCK) {
-      if (type == 'script') {
-        var surrogate = getSurrogateURI(url, requestDomain);
-        if (surrogate) {
-          return {redirectUrl: surrogate};
-        }
-      }
-
       // Notify the content script...
       var msg = {
         replaceSocialWidget: true,
         trackerDomain: requestDomain
       };
       chrome.tabs.sendMessage(tab_id, msg);
 
+      badger.checkForDNTPolicy(requestDomain);
```

Comment:
Could we make both `checkForDNTPolicy` calls in `webrequest.js` async, to avoid delaying request resolution as much as we can?   Done elsewhere in `heuristicblocking.js`:
https://github.com/EFForg/privacybadger/blob/1e3d0e748b1027b5854d06afb00237702d65bef8/src/js/heuristicblocking.js#L124-L126